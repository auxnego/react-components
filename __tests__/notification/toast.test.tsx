import React from "react";
import {create} from "react-test-renderer";

import Toast from "../../notification/toast";

it("No timer toats and open", () => {

  expect(create(
    <Toast open={true} >First Toast</Toast>
  ).toJSON()).toMatchSnapshot();

});

it("closed toast", () => {

  expect(create(
    <Toast open={false} >Never shown</Toast>
  ).toJSON()).toMatchSnapshot();

});
