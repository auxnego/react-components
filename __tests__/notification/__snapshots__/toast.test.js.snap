// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`No timer toats and open 1`] = `
<div
  className="toats"
>
  First Toast
</div>
`;

exports[`closed toast 1`] = `null`;
